build.rs

```diff
@@ -0,0 +1,57 @@
+use std::env;
+use std::path;
+use std::process;
+
+fn main() {
+    let src_dir = path::PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
+    let bpftool_dir = src_dir.join("bpftool/src");
+    let xdptools_dir = src_dir.join("xdp-tools");
+    let libxdp_dir = xdptools_dir.join("lib/libxdp");
+    let headers_dir = xdptools_dir.join("headers/xdp");
+    let include_dir = xdptools_dir.join("lib/libbpf/src/root/usr/include");
+    let libbpf_src_dir = xdptools_dir.join("lib/libbpf/src");
+
+    let status = process::Command::new("make")
+    .current_dir(&bpftool_dir)
+    .status()
+    .expect("could not execute make bpftool");
+
+    assert!(status.success(), "make failed");
+
+    let status = process::Command::new("sudo")
+    .arg("cp")
+    .arg("bpftool")
+    .arg("/usr/sbin/bpftool")
+    .current_dir(&bpftool_dir)
+    .status()
+    .expect("could not copy bpftool into /usr/sbin");
```

Comment:
This is *build*.rs, Don't copy and install bpftool here

Reply:
as your suggestion, the lines are deleted and PATH of `bpftool` is set in `build.sh`